container/ubuntu16_04/layers/bazel/version.bzl

```diff
@@ -2,5 +2,18 @@ BAZEL_VERSION_SHA256S = {
     "0.14.1": "7b14e4fc76bf85c4abf805833e99f560f124a3b96d56e0712c693e94e19d1376",
     "0.15.0": "7f6748b48a7ea6bdf00b0e1967909ce2181ebe6f377638aa454a7d09a0e3ea7b",
     "0.15.2": "13eae0f09565cf17fc1c9ce1053b9eac14c11e726a2215a79ebaf5bdbf435241",
-    "0.16.0": "bdef5499ea21baa69e707391b463105b32c4a2fbf1ab045da53f4023c1a033be",
+    "0.16.1": "17ab70344645359fd4178002f367885e9019ae7507c9c1ade8220f3628383444",
+}
+
+# This is the map from supported Bazel version to the Bazel version used to
+# generate the published toolchain configs that the former should be used with.
+# This is needed because, in most cases, patch updates in Bazel doesn't result in change
```

Comment:
do not result in changes in
(wrap line correctly)

Reply:
Done.